generators/server/templates/src/main/java/package/config/MethodSecurityConfiguration.java.ejs

```diff
@@ -19,16 +19,22 @@
 package <%= packageName %>.config;
 
 import org.springframework.context.annotation.Configuration;
+<%_ if (authenticationType !== 'oauth2') { _%>
 import org.springframework.security.access.expression.method.MethodSecurityExpressionHandler;
+<%_ } _%>
 import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
 import org.springframework.security.config.annotation.method.configuration.GlobalMethodSecurityConfiguration;
+<%_ if (authenticationType !== 'oauth2') { _%>
 import org.springframework.security.oauth2.provider.expression.OAuth2MethodSecurityExpressionHandler;
+<%_ } _%>
 
 @Configuration
 @EnableGlobalMethodSecurity(prePostEnabled = true, securedEnabled = true)
```

Comment:
Except UAA, you also need to revert to add this annotation back in the `SecurityConfiguration` class

Reply:
Sorry, a mistake on my side: what about keeping the class for oauth2 and keeping these guards?

Reply:
Unless we have a use case to override the default behavior, I don't see any value on having this additional empty configuration class.

Reply:
It's easier for the ones that wants to do it, for example, I've my own on my project:

```
private final AuthorizedOnDTOEvaluator evaluator;

    // All beans here needs to be Lazy otherwise it'll break AOP (cache, transactions, etc...)
    public MethodSecurityConfiguration(@Lazy AuthorizedOnDTOEvaluator evaluator) {
        this.evaluator = evaluator;
    }

    @Override
    protected MethodSecurityExpressionHandler createExpressionHandler() {
        return new DTOAuthorizationSecurityExpressionHandler(evaluator);
    }
```

Also, the annotation itself deserve to be in another class than the SecurityConfiguration, which ATM is dedicated to authentication and endpoint security configuration, that is external interfaces